package.json

```diff
@@ -6,7 +6,8 @@
   "main": "dist/app.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "tsc",
+    "build": "tsc && npm run copy",
+    "copy": "cp config.json dist/",
```

Comment:
This looks strange. 

COPY ../config.json . 

should work as well :-)

Reply:
Not sure what you mean